### ISSUE TITLE
ERROR Failed to read Git log: fatal: detected dubious ownership in repository

### ISSUE THREAD

**Issue by mhansen** [floryn90/docker-hugo#171]
Hi, it looks like https://github.com/floryn90/docker-hugo/issues/145 is back for me.

On this merge request: https://gitlab.com/graphviz/graphviz.gitlab.io/-/merge_requests/701/diffs

updating floryn90/hugo from 0.134.2-ext-alpine-ci -> 0.138.0-ext-alpine-ci

I now get this error: 

```
Using Docker executor with image floryn90/hugo:0.138.0-ext-alpine-ci@sha256:4749862f84e7332f71b1ad7cedafc9af6f3d595e12cda571b23a37a597b00dea ...
$ hugo
hugo: downloading modules …
hugo: collected modules in 14736 ms
Start building sites … 
hugo v0.138.0-ad82998d54b3f9f8c2741b67356813b55b3134b9+extended linux/amd64 BuildDate=[20](https://gitlab.com/graphviz/graphviz.gitlab.io/-/jobs/8292613158#L20)24-11-06T11:22:34Z VendorInfo=gohugoio
ERROR Failed to read Git log: fatal: detected dubious ownership in repository at '/builds/graphviz/graphviz.gitlab.io'
To add an exception for this directory, call:
	git config --global --add safe.directory /builds/graphviz/graphviz.gitlab.io
Total in [21](https://gitlab.com/graphviz/graphviz.gitlab.io/-/jobs/8292613158#L21)178 ms
Error: error building site: logged 1 error(s)
```

Previously this was fixed with https://github.com/floryn90/docker-hugo/pull/146, but it looks like it's back.

**Comment by mhansen** [comment]
Maybe I'm misconfiguring; should I specify a specific user?

**Comment by mhansen** [comment]
It could also be that GitLab changed something

**Comment by floryn90** [comment]
Should be solved with #172 
If the issue persists, please raise another issue and I’ll take a look into details

**Comment by mhansen** [comment]
Thank you! I’ll try it as soon as renovatebot picks up a new docker image.
